extensions/access_log_policy/plugin.cc

```diff
@@ -62,13 +62,18 @@ bool setFilterStateValue(bool log) {
 constexpr long long kDefaultLogWindowDurationNanoseconds =
     43200000000000;  // 12h
 
+constexpr StringView kRootContextId = "accesslog_inbound";
```

Comment:
Are we doing outbound access logs? Why root context ?

Reply:
No, we are not... I thought its just good practice to have it

Reply:
Talked to @bianpengyuan .. not really needed , so removing it